_drafts/2015-05-26-welcome-to-jekyll.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Seminars"
+date:   2015-05-26 12:03:10
+categories: jekyll update
```

Comment:
I haven't been using categories. I think that all they do is wind up in the URL of the post. So they only really become important when we have enough blog posts that we want to differentiate (like, `tech` posts are listed on a different page than non-tech posts).